docs/scripts_md/concat_mri.md

```diff
@@ -0,0 +1,62 @@
+# NAME
+concat\_mri.pl - aggregates minc files that have a common set of properties.
+
+# SYNOPSIS
+concat\_mri.pl \[options\] \[minc\_files\]
+
+- **-clean** : remove the original files when they are concatenated
+- **-clobber** : overwrite existing output file(s)
+- **-debug** : print commands but do not execute them
+- **-verbose** : print additional information while running
+- **-compress** : compress resulting output files with `gzip`
+- **-stdin** : do not read the names of the minc files on the commmand line but read them
+from STDIN instead
+- **-postfix \[postfix\]** : Create output file names using the first file name, followed by 
+`postfix`. `postfix` will be inserted before the `.mnc` extension. If this option is not
+used, files will be created as if `-postfix _concat` was used.
```

Comment:
do I understand correctly? If -postfix is not mentioned in the command line, it will add "_concat" to the filename before the `.mnc`. If -postfix [postfix] is specified, then it would use whatever string is in [postfix] and append it to the filename before the `.mnc`?

Might be worth clarifying that `postfix` in the text refer to what is in [] (or maybe simply add the [] around postfix?

Hope that makes sense.

Reply:
Changed the text so it is easier to understand. Also removed the [] around the postfix argument.